server/lib/resolvers/builders/nedb.js

```diff
@@ -48,6 +49,7 @@ function buildNeDBResolver (dataSource, compiledRequestMapping, compiledResponse
 
       function mapResponse (err, res) {
         if (err) return reject(err)
+        if (!res) return resolve()
```

Comment:
@aliok should not the linter complain about this point either?

Reply:
@camilamacedo86 I rebased this branch on top of master (has now linter config fixed)  *locally* and it didn't complain about that line. What's the problem there? BTW, we're using this format: https://standardjs.com